learning-development/accessibility.qmd

```diff
@@ -52,6 +51,48 @@ For more advice about colour in charts and visualisations specifically, includin
 
 ---
 
+## Making spreadsheets accessible
+
+When creating an accessible spreadsheet, it’s important to consider the needs of all users, including those with disabilities. To support analysts in making spreadsheets more accessible, here are five simple yet effective strategies:
+
+### a. Consult the Spreadsheet Accessibility Guidance
```

Comment:
I'd remove the "a." and change to sentence case:
"Consult the spreadsheet accessibility guidance"